src/provisioner/provisioner.py

```diff
@@ -199,4 +191,44 @@ def cluster_bootstrap(cortx_conf_url: str = None):
                         rc, "%s phase of %s, failed. %s", interface,
                         components[comp_idx]['name'], err)
 
-        Log.info(f'Finished cluster bootstrap on {node_id}:{node_name}')
+    @staticmethod
+    def cluster_bootstrap(cortx_conf_url: str = None):
+        """
+        Description:
+        Configures Cluster Components
+        1. Reads Cortx Config and obtain cluster components
+        2. Invoke Mini Provisioners of cluster components
+        Paramaters:
+        [IN] CORTX Config URL
+        """
+        if cortx_conf_url is None:
```

Comment:
Why do you need to handle None case ? Why dont we make this parameter mandatory ?

Reply:
made mandatory param